dotcom-rendering/src/web/components/Footer.tsx

```diff
@@ -198,7 +200,7 @@ const FooterLinks = ({
 	contributionsServiceUrl: string;
 }) => {
 	const linkGroups = pageFooter.footerLinks.map((linkGroup) => {
-		const linkList = linkGroup.map((l: FooterLink, index: number) => (
+		const linkList = linkGroup.map((l, index) => (
```

Comment:
Removal of types, which can be automatically inferred.

Reply:
nice!